src/app/modules/core-bcp/models/helperFunc.spec.ts

```diff
@@ -0,0 +1,62 @@
+import {setNotApplicable, formatDateForDisplay, convertToJSONDate, stripPhoneFormatting, stripPostalCodeSpaces} from './helperFunc';
+
+describe('setNotApplicable', () => {
+    it('should return value when value passed is defined', () => {
+        expect(setNotApplicable("test")).toEqual("test");
+    });
+
+    it('should return N/A for undefined value', () => {
+        expect(setNotApplicable(undefined)).toEqual("N/A");
+        expect(setNotApplicable(null)).toEqual("N/A");
+    });
+});
+
+
+describe('formatDateForDisplay', () => {
+    it('should return date in readable format', () => {
+        expect(formatDateForDisplay(new Date("January 1, 2019"))).toEqual("January 01, 2019");
+        expect(formatDateForDisplay(new Date("December 1, 2019"))).toEqual("December 01, 2019");
```

Comment:
This is good, but add a couple more values just to make sure.

For example, make sure to add a date that isn't the 1st of a month (e.g. does doing "January 10th" lead to "Jan 010"?).  I would even add a leap year to make sure.

Reply:
Done.